lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -31,27 +31,19 @@ def format
 SYNOPSIS
 <% synopses.each do |s| %>
     <%= s %>
-<% end %>
-<% unless @command.long_description.nil? %>
+<% end %><% unless @command.long_description.nil? %>
 
 DESCRIPTION
     <%= wrapper.wrap(@command.long_description) %> 
-<% end %>
-<% if options_description.strip.length != 0 %>
-
+<% end %><% if options_description.strip.length != 0 %>
 COMMAND OPTIONS
 <%= options_description %>
-<% end %>
-<% unless @command.commands.empty? %>
-
+<% end %><% unless @command.commands.empty? %>
```

Comment:
are these meant to reduce some of the extra whitespace that can happen in the autogenerated help?

I tried doing a little fix on this locally w/ the ERB trim mode and ` -%>` , but got lost in the intricacies of ERB and removing whitespace. 🤯

Reply:
Yes, I also tried that but it didn't immediately work so I just joined up the lines.  Not the most elegant solution but it does seem to work :)

Reply:
👍